tests/onnxruntime/test_modeling.py

```diff
@@ -3082,6 +3082,21 @@ class ORTModelForSeq2SeqLMIntegrationTest(ORTModelTestMixin):
     GENERATION_LENGTH = 100
     SPEEDUP_CACHE = 1.1
 
+    def _get_model_ids(self, model_arch):
+        model_ids = MODEL_NAMES[model_arch]
+        if isinstance(model_ids, dict):
+            model_ids = list(model_ids.keys())
+        else:
+            model_ids = [model_ids]
+        return model_ids
```

Comment:
So what you call `model_arch` is a "model type" like `bert`, `t5` and so on?
This function returns a list of model ids from the Hub for a given architecture?

Reply:
It's a helper function for the test to get the list of `model_ids` for a given `model_arch(model_type)` from `MODEL_NAMES`. Earlier there was one `model_id` for a `model_arch`. But with this PR there can be multiple (Format is same as used in onnx export).